spec/models/manageiq/providers/inventory/persister/targeted_refresh_spec_helper.rb

```diff
@@ -7,7 +7,7 @@ def create_persister
   end
 
   def expected_ext_management_systems_count
-    2
+    3
```

Comment:
haha, so obvious

Reply:
well it's the same as https://github.com/ManageIQ/manageiq/pull/18842/files#r293334487 so yes.

Reply:
yes, I was being sarcastic... it was an existing test... I'm being silly